### PR TITLE
Change point outline api to integrate with the latest tangram release

### LINKED ISSUE #32: [Tangram R&D] Update API to make it compatible with 0.12.0 tangram release
The new tangram point outline feature has a different API.

It is:
```javascript
{ 
  outline: { 
     width: "5px",
     color: red
  }
}
```

We had:
```javascript
{ 
  outline_width: "5px",
  outline_color: red
}
```

### PR DESCRIPTION
**Pull request by donflopez**
Fix #32

### CODE REVIEW

File: src/basic/points.js [hunk 1]
```diff
@@ -114,10 +118,10 @@ Point.getDraw = function(c3ss, id) {
 
 	if (checkMarkerSym(c3ss)) {
 
-		Object.assign(
-				point,
-				getColors(c3ss),
-				getWidths(c3ss)
+		point = mergeWith(
```
Comment (dgaubert) [inline src/basic/points.js]: Just asking: What are the benefits of using `mergeWith/merge` instead of de-facto function `Object.assign`?

Comment (donflopez) [inline src/basic/points.js]: It is intended for having a decision function for conflicts...

Here both, `getColors` and `getWidths` return and object with an equal property wich others, this property is outline, so if outline already exists I use merge again to have a deep merge.

With `Object.assing` we have the latest outline, not a merged outline.

Comment (dgaubert) [inline src/basic/points.js]: Cool 😎  , thanks for answering!

**Comment by dgaubert** [comment]
64px-bandera_de_jaen svg

**Pull request merged**